perftest/simulations/src/test/scala/org/projectnessie/perftest/gatling/CommitToBranchSimulation.scala

```diff
@@ -51,33 +51,28 @@ class CommitToBranchSimulation extends Simulation {
           val tableName = params.makeTableName(session)
 
           // Call the Nessie client operation to perform a commit
-          val key = ContentKey.of("name", "space", tableName)
-          val contentId = tableName
+          val key = ContentKey.of("name", "space", tableName, userId.toString)
+          val contentId = tableName + "_" + userId.toString
 
           val tableMeta = IcebergTable
             .of(
-              s"path_on_disk_${tableName}_$commitNum",
+              s"path_on_disk_${tableName}_${userId}_$commitNum",
               42,
               43,
               44,
               45,
               contentId
             )
 
-          // TODO the expectedContent is wrong!! commitNum is for the session, but we need the actual global state!!
+          val globalStates = client.getContent.reference(branch).key(key).get()
```

Comment:
it's not "global", it's rather just existing content

Reply:
the existing `TODO` states that we should use the `global state`, that is why I named it this way. Isn't it global?

Reply:
let's just change this to `val existingTable = client.getContent.reference(branch).key(key).get().get(key)`